tests/e2e/data/services/test_bq_datasets.py

```diff
@@ -135,3 +135,30 @@ def test_creation_of_dataset(self):
         df = pandas.read_csv(result)
         self.assertEqual(df.shape, (0, 2))
         self.assertEqual(df.to_csv(index=False), 'cartodb_id,the_geom\n')
+
+    def test_enrichment_of_dataset(self):
+        unique_table_name = 'cf_test_table_' + str(uuid.uuid4()).replace('-', '_')
+        sample = StringIO(CSV_SAMPLE_REDUCED)
+        df = pandas.read_csv(sample)
+
+        dataset = BQUserDataset \
+            .name(unique_table_name) \
+            .column(name='cartodb_id', type='INT64') \
```

Comment:
To avoid collisions, we are using `__enrichment_id` and `__geom_column` as column names.